modules/ROOT/pages/runtime-configure-eu.adoc

```diff
@@ -1,19 +1,19 @@
-= To Configure the Mule Runtime to Connect to the EU Control Plane
+= Configure Mule runtime engine to Connect to the EU Control Plane
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To use Mule Runtime within the EU control plane, you must redefine the configuration properties that enable Mule Runtime to point to the EU control plane. You must also provide the client ID and secret of your environment.
+To use Mule runtime engine within the EU Control Plane, you must redefine the configuration properties that enable Mule runtime engine to point to the EU Control Plane. You must also provide the client ID and client secret of the environment belonging to your organization.
 
-By default, the Mule Runtime is configured to point to the US control plane. If you do not override these properties, the Mule Runtime modules are disabled in the EU control plane.
+By default, Mule runtime engine is configured to point to the US control plane. If you do not override these properties, the API Gateway capability of Mule runtime engine is disabled in the EU Control Plane.
 
-The properties you must override depend on which version of the Mule runtime you are using, as described in the following sections.
+The properties you must override depend on the version of Mule runtime engine you are using, as described in the following sections.
 
-== Configuring Mule Runtime 4.0
+== Configuring Mule 4.0
 
```

Comment:
GLOBALLY rework text to not have contiguous titles or headings. Insert something that, at a minimum, prepares the audience for the subtopics within a container.

Reply:
Suggest something like...:

"You can configure Mule runtime engine (Mule) using `wrapper.conf`, the command line, or CloudHub."

If there is anything to say about under what circumstances a user might choose one method over another, suggest providing that here as well.

Reply:
Great - thank you!